subprojects/delivery/nupokati/src/main/kotlin/com/avito/android/ArtifactoryExtension.kt

```diff
@@ -0,0 +1,24 @@
+package com.avito.android
+
+import org.gradle.api.provider.Property
+
+public abstract class ArtifactoryExtension {
+
+    public abstract val baseUrl: Property<String>
+
+    public abstract val login: Property<String>
+
+    public abstract val password: Property<String>
+
+    /**
+     * This parameters used to create path in artifactory repository
+     * and differs from one that comes from json cd build config
+     * There is no real reason of that
+     *
+     *  todo merge, take one from json
```

Comment:
That should be fixed without any changes in service contract, but it's a risky one, because we will start to write files in another directory/repository in artifactory.
There is a possibility that someone already used that paths, but a small one, because it's designed to be used only by service itself

I will do a change, on next phase
All old artifacts can't be moved without changing result json's accordingly, so maybe i should not bother